.env.test

```diff
@@ -0,0 +1,5 @@
+KERNEL_CLASS='AppKernel'
+APP_SECRET='$ecretf0rt3st'
+SYMFONY_DEPRECATIONS_HELPER=weak
+PANTHER_APP_ENV=panther
```

Comment:
`symfony\panther` is not used in our project

Reply:
@nicosomb I know, I kept it to be as close as possible to the recipe. Maybe one day it will be used for e2e tests. And as it's a env var I don't think it hurts having it.
Another option would be to make those a comment like TRUSTED_HOSTS, what do you think?
If not OK I can of course remove them :)